data/json/requirements/materials.json

```diff
@@ -125,105 +144,135 @@
-    "components": [ [ [ "leather", 6 ], [ "tanned_hide", 1 ] ] ]
+    "//": "Materials used for crafting leather clothing.",
+    "components": [ [ [ "leather", 8 ], [ "sheet_leather_patchwork", 1 ], [ "tanned_hide", 1 ] ] ]
   },
   {
     "id": "fabric_leather_fur_hide",
     "type": "requirement",
-    "//": "Materials used for crafting leather clothing where fur is or should be an option (can be scraped off with a cutting tool for example), when tanned hides/pelts are available it seems to roughly be a 1:6 ratio",
-    "components": [ [ [ "leather", 6 ], [ "fur", 6 ], [ "tanned_hide", 1 ], [ "tanned_pelt", 1 ] ] ]
+    "//": "Materials used for crafting leather clothing where fur is or should be an option (can be scraped off with a cutting tool for example), when tanned hides/pelts are available it seems to roughly be a 1:8 ratio",
+    "components": [
+      [
+        [ "leather", 8 ],
+        [ "fur", 8 ],
+        [ "tanned_hide", 1 ],
+        [ "tanned_pelt", 1 ],
+        [ "sheet_leather_patchwork", 1 ],
+        [ "sheet_fur_patchwork", 1 ]
+      ]
+    ]
   },
   {
     "id": "fabric_nomex",
     "type": "requirement",
     "//": "Materials used for crafting Nomex clothing",
-    "components": [ [ [ "nomex", 1 ], [ "sheet_nomex_patchwork", 1 ], [ "sheet_nomex", 1 ] ] ]
+    "components": [ [ [ "nomex", 8 ], [ "sheet_nomex_patchwork", 1 ], [ "sheet_nomex", 1 ] ] ]
   },
   {
     "id": "fabric_nylon",
     "type": "requirement",
     "//": "Materials used for crafting nylon clothing",
-    "components": [ [ [ "nylon", 1 ], [ "sheet_nylon_patchwork", 1 ], [ "sheet_nylon", 1 ] ] ]
+    "components": [ [ [ "nylon", 8 ], [ "sheet_nylon_patchwork", 1 ], [ "sheet_nylon", 1 ] ] ]
   },
   {
     "id": "fabric_neoprene",
     "type": "requirement",
     "//": "Materials used for crafting fur clothing",
-    "components": [ [ [ "neoprene", 1 ], [ "sheet_neoprene_patchwork", 1 ], [ "sheet_neoprene", 1 ] ] ]
+    "components": [ [ [ "neoprene", 8 ], [ "sheet_neoprene_patchwork", 1 ], [ "sheet_neoprene", 1 ] ] ]
   },
   {
     "id": "fabric_lycra",
     "type": "requirement",
     "//": "Materials used for crafting lycra clothing",
-    "components": [ [ [ "lycra_patch", 1 ], [ "sheet_lycra", 1 ], [ "sheet_lycra_patchwork", 1 ] ] ]
+    "components": [ [ [ "lycra_patch", 8 ], [ "sheet_lycra", 1 ], [ "sheet_lycra_patchwork", 1 ] ] ]
   },
   {
     "id": "filament",
     "type": "requirement",
-    "//": "Materials for use when sewing items",
+    "//": "Materials for use when sewing items. 10 inches of thread, sinew, fiber or yarn, or 12 inches of nomex and kevlar thread per unit.",
     "components": [
-      [ [ "thread", 1 ], [ "sinew", 1 ], [ "plant_fibre", 1 ], [ "yarn", 1 ], [ "thread_nomex", 1 ], [ "thread_kevlar", 1 ] ]
+      [ [ "thread", 1 ], [ "sinew", 1 ], [ "plant_fibre", 1 ], [ "yarn", 1 ], [ "thread_nomex", 2 ], [ "thread_kevlar", 2 ] ]
```

Comment:
Grass Yarn should probably be part of the "filament" requirements group as well.

Reply:
the description of grass yarn specifically states that it’s “too thick to sew with”. 

I could see some realism bending for innawoods, but that would be better suited overriding the group in the mod, rather than in the base game.